sagemaker/launch/common_parser.py

```diff
@@ -5,6 +5,10 @@
 from ast import literal_eval
 import argparse
 
+# Npte: Sync with graphstorm.config SUPPORTED_TASKS
```

Comment:
Note：

Reply:
Btw, why not directly import from graphstorm.config

Reply:
The launch script will not rely on graphstorm package.